ui/partials/perf/testdatachooser.html

```diff
@@ -3,19 +3,46 @@
   <h3 class="modal-title">Add test data</h3>
 </div>
 <div class="modal-body">
-  <h4>Project</h4>
-  <select class="form-control" ng-change="updateTestInput()" ng-model="selectedProject" ng-options="project.name for project in projects">
-  </select>
-  <h4>Platform</h4>
-  <p class="blink" ng-show="loadingTestData">Loading series data...</p>
-  <select ng-hide="loadingTestData" ng-change="updateTestSelector()" ng-model="selectedPlatform" ng-options="platform for platform in platformList">
-  </select>
-  <h4>Test</h4>
-  <p class="blink" ng-show="loadingTestData">Loading series data...</p>
-  <div id="scrollable-dropdown-menu">
-    <input my-input ng-hide="loadingTestData" class="typeahead" type="text" placeholder="Enter test keywords (e.g. 'tp5o')"></input>
+  <div class="test-chooser">
+    <h4>Project</h4>
+    <select class="form-control" ng-change="updateTestInput()" ng-model="selectedProject" ng-options="project.name for project in projects">
+    </select>
+    <h4>Platform</h4>
+    <p class="blink" ng-show="loadingTestData">Loading series data...</p>
+    <select ng-hide="loadingTestData" ng-change="updateTestSelector()" ng-model="selectedPlatform" ng-options="platform for platform in platformList">
+    </select>
+    <h4>Tests</h4>
+    <div class="form-group">
+        <input class="form-control input-sm" type="text" ng-model="testFilter" placeholder="Filter tests"/>
```

Comment:
Looking at other Perf partials, it seems Perf is using 2 space indent for html also. I think any 4 space html indent in this PR like the line above, can safely be switched to 2. Same with occurrences below.